models/offer_proposition.py

```diff
@@ -67,7 +67,8 @@ class OfferProposition(SerializableModel):
 
     @property
     def recent_acronym_education_group(self):
-        return self.education_group.most_recent_acronym
+        if self.education_group:
+            return self.education_group.most_recent_acronym
```

Comment:
Il ne faut rien retourner si l'offer_proposition n'a pas d'education_group ? Est-ce que c'est un cas business logique ?

Reply:
oui oui c'est normal, la fonction ne retourne pas rien mais None, si non ça plante. erreur Object is None